CMakeLists.txt

```diff
@@ -401,8 +415,22 @@ if(EMSCRIPTEN)
     xeus_cpp_set_kernel_options(xcpp)
     xeus_wasm_compile_options(xcpp)
     xeus_wasm_link_options(xcpp "web,worker")
+    target_link_options(xcpp PUBLIC
+        -sEXPORTED_RUNTIME_METHODS=FS,PATH,ERRNO_CODES
```

Comment:
This would also be removed once we update to xeus 6 (a migration is planned soon)

Check https://github.com/jupyter-xeus/xeus/pull/412

Reply:
Add a comment in the code saying that.